assembl/graphql/schema.py

```diff
@@ -480,13 +511,17 @@ def resolve_order(self, args, context, info):
 class Query(graphene.ObjectType):
     node = Node.Field()
     posts = SQLAlchemyConnectionField(PostConnection, idea_id=graphene.ID())
-    ideas = SQLAlchemyConnectionField(Idea)
+    root_idea = graphene.Field(Idea)
+    ideas = graphene.List(Idea)
```

Comment:
ideas query results contain all ideas, so in the browser you need to reconstruct a tree based on parentId (root idea has parentId None)